pkg/operators/localmanager/localmanager.go

```diff
@@ -54,6 +54,7 @@ const (
 	PodmanSocketPath     = "podman-socketpath"
 	ContainerdNamespace  = "containerd-namespace"
 	RuntimeProtocol      = "runtime-protocol"
+	WithK8sApiserver     = "with-k8s-apiserver"
```

Comment:
I'm not sure about the naming, it doesn't have the world enrich/enrichment and it's all it does. `enrich-with-k8s-apiserver` or maybe just `enrich-with-k8s`?

Reply:
I would go with `enrich-with-k8s-apiserver` then.

Since we already have K8s information through labels. Then having an additional flag called `enrich-with-k8s` would confuse a user IMO